lms/djangoapps/certificates/views/webview.py

```diff
@@ -53,10 +53,11 @@ def get_certificate_description(mode, certificate_type, platform_name):
     """
     certificate_type_description = None
     if mode == 'honor':
-        # Translators:  This text describes the 'Honor' course certificate type.
-        certificate_type_description = _("An {cert_type} Certificate signifies that an {platform_name} "
-                                         "learner has agreed to abide by {platform_name}'s honor code and "
-                                         "completed all of the required tasks for this course under its "
+        # Translators:  This text describes the 'Honor' course
+        # certificate type.
```

Comment:
@mattdrayer I ran the PEP8 test, which complained about that line length:

vagrant@precise64:/edx/app/edxapp/edx-platform$ pep8 lms/djangoapps/certificates/views/webview.py
lms/djangoapps/certificates/views/webview.py:56:80: E501 line too long (80 characters)

I found that that test will continue complaining about lines being too long, one line at a time. So I can rejoin this line.